docs/source/upcoming_release_notes/6-Added_argparse_module_and_fixed_bugs.rst

```diff
@@ -0,0 +1,23 @@
+6 Added_argparse_module_and_fixed_bugs
+#################
+
+API Breaks
+----------
+- N/A
+
+Features
+--------
+- N/A
+
+Bugfixes
+--------
+Added argparse module to handle CLI tool. Fixed various bugs.
+
+
+Maintenance
+-----------
+- N/A
+
+Contributors
+------------
+Jane Liu
```

Comment:
```suggestion
- janeliu-slac
```

This should be your github username (we should really make this more obvious somehow)